### DISCUSSION TITLE
MISP templates

### DISCUSSION THREAD

**Discussion by joash2808** [MISP/PyMISP#845] [Q&A]
How do you make misp templates? sorry the docs is a bit vague

**Comment by Rafiot** [comment]
I'm not sure what you mean. Which templates are you talking about?

**Reply by joash2808** [reply]
hmmm so I am trying to see if I can develop something such that when the user wants to key in the attributes, it can help standardise the inputs and make sure the user key in the needed fields. (trying to find a way for people in the organisation to standardise what they key in)  I am not sure if misp have any way to implement this? I am also not sure what does the templates do in the misp platform. Am hoping a kind soul, could help break down what does the templates on misp platform do. I see there is a "add template" button that can be found in the "event actions".  I have read the docs but I still do not quite understand this part.

**Reply by Rafiot** [reply]
Oh, right. I don't think the templating system is accessible via the API. Checking it with @iglocska and @mokaddem.
